tests/pytestHelper.py

```diff
@@ -0,0 +1,25 @@
+import os
+import glob
+import subprocess
+
+testDir = os.path.abspath(os.path.dirname(__file__))
+testInputsDir = os.path.join(testDir, 'test_inputs')
+testOutputsDir = os.path.join(testDir, 'test_outputs')
+sampleBcDir = os.path.join(testOutputsDir, 'sampleBC')
+sleipnirDir = os.path.dirname(testDir)
+sleipnirBin = os.path.join(sleipnirDir, 'Debug')
+seekScriptsDir = os.path.join(sleipnirDir, 'scripts', 'seek')
+
+def createSampleDatabase():
+    dbDir = os.path.join(sampleBcDir, 'db')
+    dbFiles = glob.glob1(dbDir, '*.db')
+    if len(dbFiles) < 100:
+        os.makedirs(sampleBcDir, exist_ok=True)
+        inputBCFiles = os.path.join(testInputsDir, 'breast-cancer-sample')
+        os.system(f'cp -a {inputBCFiles}/* {sampleBcDir}/')
```

Comment:
Tiny thing - I see some inconsistency with BC abbreviation, once it is `inputBCFiles`, the other time `sampleBcDir`

Reply:
Ok, I improved the variable name and added a comment. This is where the "reference" breast cancer sample files (such as PCL files) are copied from the checked-in input directory to a temporary output directory in preparation for building the database.